### DISCUSSION TITLE
Should we allow a user-defined LS factor in SDR?

### DISCUSSION THREAD

**Discussion by phargogh** [natcap/invest#1300] [Ideas]
[A user on the forums had this suggestion](https://community.naturalcapitalproject.org/t/new-invest-workbench-beta-release/2324/8?u=jdouglass):

> Dear InVEST developers
>
> The new interface of the model is looking great. However, as I suggested in my previous comment, the LS factor in the SDR sub-model is still can not be customized. In other words, users are not allowed to calculate the LS factor with another formula. My strong suggestion is the LS factor in the SDR sub-model should be put like the advanced and default options of the seasonal water yield sub-model. In general, two options (i.e., one user-defined and another with default) are needed for calculating the LS factor in the SDR sub-model. I hope that my suggestions will be considered in the next version of the model.

I don't recall the details of our LS factor implementation, but this sounds like a reasonable request and there may be some precedent for this in some of the other hydro models.  Either way, it'd be helpful to record our decision about this in this issue thread.

**Comment by emlys** [comment]
We should talk with a hydrologist in order to make a decision about this.

**Comment by phargogh** [comment]
Some of this will probably be addressed by the updated LS factor #915 since our LS Factor will be more in line with what people expect.  On the other hand, this user is requesting the ability to use different LS factor equation, which the improved LS factor output in InVEST would not address.
